auth/src/main/res/values-zh-rTW/strings.xml

```diff
@@ -47,7 +47,6 @@
     <string name="fui_error_unknown">發生不明錯誤。</string>
     <string name="fui_error_invalid_password">密碼不正確。</string>
     <string name="fui_accessibility_logo">應用程式標誌</string>
-    <string name="fui_signed_in_with_specific_account">以 %1$s 的身分登入</string>
```

Comment:
Thanks for reverting the `tools` declaration, but can we just revert all changes to the `strings` files?  Is there some lint about an unused resource you're trying to get around?

I would much rather ignore the lint for now than have all of these strings changes cluttering up this very important PR.

Reply:
Oh gotya, I'll tell lint to keep calm and let the resources be unused. 😆